tracex/extraction/logic/orchestrator.py

```diff
@@ -80,6 +87,10 @@ def initialize_modules(self):
             self.configuration.modules["time_extraction"](),
             self.configuration.modules["event_type_classification"](),
             self.configuration.modules["location_extraction"](),
+            # This module should be activated only if the user wants to analyze the metrics
+            # self.configuration.modules["metrics_analyzer"](),
+            # Only activate this module with a test comparison patient journey as ground truth
+            # self.configuration.modules["event_log_comparator"](),
```

Comment:
Maybe we can tie this to the user input in the UI in the future. Something like a "track metrics" check box. The `ExtractionConfiguration` class would then need to be initialized with or without the metrics modules depending on that choice. Maybe not now, but should keep that in mind I think.

Reply:
sounds good!